scripts/azure_logs_integration_original_DD.js

```diff
@@ -162,20 +159,20 @@ class Batcher {
   }
 }
 
+// Provide Kloudfuse URL in quotes for "hostname" where you want to send the logs to
 class HTTPClient {
   constructor(context) {
     this.context = context;
     this.httpOptions = {
-      hostname: DD_HTTP_URL,
-      port: DD_HTTP_PORT,
+      hostname: "<Kloudfuse URL>",
```

Comment:
Introduce a variable for this as well so its easier to set it.

Reply:
okay